### PR TITLE
Make individual request for each request in batch

### PR DESCRIPTION
**Pull request by ternovoy**
@alexeytokar @rasmus93

### CODE REVIEW

File: src/main/java/ua/net/tokar/json/rainbowrest/RainbowRestOncePerRequestFilter.java [hunk 1]
```diff
@@ -57,6 +71,37 @@ protected String getResponseViaInternalDispatching(
         return responseWrapper.getCaptureAsString();
     }
 
+    protected String getResponseViaInternalDispatching(
+            URI uri,
+            Header[] headers
+    ) throws ServletException, IOException, URISyntaxException {
+        try ( CloseableHttpClient httpClient = HttpClients.createDefault() ) {
+            HttpGet httpget = new HttpGet( uri );
```
Comment (rasmus93) [inline src/main/java/ua/net/tokar/json/rainbowrest/RainbowRestOncePerRequestFilter.java]: `httpGet`

File: src/main/java/ua/net/tokar/json/rainbowrest/RainbowRestOncePerRequestFilter.java [hunk 2]
```diff
@@ -57,6 +71,37 @@ protected String getResponseViaInternalDispatching(
...
+    }
+
+    protected Header[] getHeaders( HttpServletRequest request ) {
+        List<Header> headers = new ArrayList<>();
+        for ( Enumeration<String> e = request.getHeaderNames(); e.hasMoreElements(); ) {
+            String name = e.nextElement();
+            headers.add( new BasicHeader( name, request.getHeader( name ) ) );
+        }
+        Header[] headersArray = new Header[headers.size()];
+        return headers.toArray( headersArray );
```
Comment (rasmus93) [inline src/main/java/ua/net/tokar/json/rainbowrest/RainbowRestOncePerRequestFilter.java]: remove array, i.e. `header.toArray( new Header[headers.size()] );`

**Comment by alexeytokar** [comment]
Please describe the reason for the change

**Comment by ternovoy** [comment]
There is a problem with making copy of request and response. As a result client throw java.lang.IllegalStateException: Cannot forward after response has been committed

**Comment by ternovoy** [comment]
fixed

**Pull request merged**